package.json

```diff
@@ -222,8 +222,8 @@
     "sinon": "^7.3.2",
     "sinon-chai": "^3.3.0",
     "style-loader": "^0.13.1",
-    "stylelint": "^12.0.0",
-    "stylelint-scss": "^2.1.0",
+    "stylelint": "^13.0.0",
```

Comment:
Breaking change between 12.x.x and 13.x.x is dropping node 8, which doesn't affect us.